webpack.config.js

```diff
@@ -26,14 +26,19 @@ const config = [
       bundle: `${PATHS.SRC}/bundles/bundle.js`,
       // See https://github.com/webpack/webpack/issues/300#issuecomment-45313650
       SilverStripeNavigator: `${PATHS.LEGACY_SRC}/SilverStripeNavigator.js`,
+      'TinyMCE_sslink-internal': `${PATHS.LEGACY_SRC}/TinyMCE_sslink-internal.js`,
     },
     output: {
       path: PATHS.DIST,
       filename: 'js/[name].js',
     },
     devtool: (ENV !== 'production') ? 'source-map' : '',
     resolve: resolveJS(ENV, PATHS),
-    externals: externalJS(ENV, PATHS),
+    externals: Object.assign({}, externalJS(ENV, PATHS), {
+      'containers/InsertLinkModal/InsertLinkModal': 'InsertLinkModal',
```

Comment:
This can be removed after https://github.com/silverstripe/webpack-config/pull/2 is on NPM.